src/ResourceManager/Sql/Commands.Sql/ChangeLog.md

```diff
@@ -23,6 +23,9 @@
 * Clarified User-Defined Restore Points for SQLDW in New-AzureRmSqlDatabaseRestorePoint help
 * Fixed formatting of OutputType in help files
 * Updated documentation of -ComputeGeneration parameter in several cmdlets
+* Adding new Cmdlets for Management.Sql to allow customers to add TDE Certificate to Sql Server instance or a Managed Instance
```

Comment:
@nivimsft since we pulled in the latest changes from our release, you'll need to move the snippet you added in this file to be under `## Current Release`